SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/ClientServerRPCService.cpp

```diff
@@ -219,6 +225,12 @@ void ClientServerRPCService::OnEndpointAuthorityGained(const Worker_EntityId Ent
 		LastAckedRPCIds.Add(EntityRPCType(EntityId, ERPCType::ServerUnreliable), Endpoint.UnreliableRPCAck);
 		RPCStore->LastSentRPCIds.Add(EntityRPCType(EntityId, ERPCType::ClientReliable), Endpoint.ReliableRPCBuffer.LastSentRPCId);
 		RPCStore->LastSentRPCIds.Add(EntityRPCType(EntityId, ERPCType::ClientUnreliable), Endpoint.UnreliableRPCBuffer.LastSentRPCId);
+
+		// Ensure that any RPCs pending in this entity are executed.
+		for (ERPCType Type : { ERPCType::ServerReliable, ERPCType::ServerUnreliable })
```

Comment:
Clients should never need to do this?

Reply:
What if a server creates a player controller and immediately executes a client rpc? Seems valid. Obviously needs to be in the section above though.